src/ServicePulse.Host/vue/src/components/failedmessages/EditRetryDialog.vue

```diff
@@ -0,0 +1,87 @@
+<script setup>
+import { reactive, onMounted } from "vue";
+
+const emit = defineEmits(["cancel"]);
+
+const settings = defineProps({
+  message: Object,
+});
+
+const panel = reactive({ number: undefined });
+
+/* function retry() {
+    emit("retry", settings);
+} */
+
+function close() {
+  emit("cancel");
+}
+
+function togglePanel(panelNum) {
+  panel.number = panelNum;
+  return false;
+}
+
+onMounted(() => {
+  togglePanel(1);
+});
+</script>
+
+<template>
+  <div class="container">
+    <section name="failed_message">
+      <div class="modal-mask">
+        <div class="modal-content">
+          <div class="modal-container">
+            <div class="modal-header">
+              <div class="modal-title">
+                <h3>Edit and retry message</h3>
+              </div>
+            </div>
+            <div class="modal-body">
+              <div class="row">
+                <div isolate-click class="col-sm-12 no-side-padding">
```

Comment:
```suggestion
                <div class="col-sm-12 no-side-padding">
```
`isolate-click` is an angular directive that does nothing here, it can be deleted.